plugins/woocommerce/src/Internal/Admin/ShippingLabelBanner.php

```diff
@@ -60,7 +60,9 @@ private function should_show_meta_box() {
 			$incompatible_plugins = class_exists( '\WC_Shipping_Fedex_Init' ) ||
 				class_exists( '\WC_Shipping_UPS_Init' ) ||
 				class_exists( '\WC_Integration_ShippingEasy' ) ||
-				class_exists( '\WC_ShipStation_Integration' );
+				class_exists( '\WC_ShipStation_Integration' ) ||
+				class_exists( '\Automattic\WCShipping\Loader' ) ||
+				class_exists( '\Automattic\WCTax\Loader' );
```

Comment:
This prevents the following banner from being displayed on the Edit order page when either WC Shipping or WC Tax are present:

![image](https://github.com/woocommerce/woocommerce/assets/1759681/340d07e8-ac42-4151-8432-bc68a41fb88a)